fixup

```diff
@@ -0,0 +1,11 @@
+cat >dist/cjs/package.json <<!EOF
```

Comment:
Not major but this should probably have a shebang line as this script doesn't work in my shell
```suggestion
#!/bin/bash
cat >dist/cjs/package.json <<!EOF
```